### PR TITLE
HIPAA Rules in test

### PR DESCRIPTION
**Pull request by anivan-suse**
#### Description:

- grub2_disable_interactive_boot
- ensure_suse_gpgkey_installed

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml [hunk 1]
```diff
@@ -1,9 +1,21 @@
-# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
+# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
```
Comment (vojtapolasek) [inline linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml:1]:
```suggestion
# platform = multi_platform_all
```

File: linux_os/guide/system/software/updating/ensure_gpgcheck_local_packages/ansible/shared.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/updating/ensure_gpgcheck_local_packages/ansible/shared.yml:1]:
```suggestion
# platform = multi_platform_all
```

**Review by vojtapolasek — DISMISSED** [review]
Hello, LGTM, just two small suggestions.

File: linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
 # reboot = true
 # strategy = restrict
 # complexity = low
 # disruption = low
 
+{{%if 'sle' in product %}}
```
Comment (ggbecker) [inline linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml:7]:
```suggestion
{{% if 'sle' in product %}}
```

**Review by ggbecker — APPROVED** [review]
Tested on RHEL8 environment and both ansible and bash remediation pass the SSGTS tests.

**Pull request merged**